### PR TITLE
Create new certificate

### PR DESCRIPTION
**Pull request by KyleMacPherson**
https://uktrade.atlassian.net/jira/software/projects/TARIFFS/boards/127?selectedIssue=TARIFFS-494

### CODE REVIEW

File: app/helpers/approval_helper.rb [hunk 1]
```diff
@@ -54,6 +54,8 @@ def approval_table_partial(workbasket)
       "workbaskets/shared/steps/review_and_submit/approval/edit_footnote"
     elsif workbasket.type == 'edit_regulation'
       "workbaskets/shared/steps/review_and_submit/approval/edit_regulation"
+    elsif workbasket.type == 'create_certificate'
+      "workbaskets/shared/steps/review_and_submit/approval/create_certificate"
     end
   end
 end
```
Comment (steveholmes) [inline app/helpers/approval_helper.rb]: Does the approval_title method need a new elsif as well?

File: app/helpers/cross_check_helper.rb [hunk 2]
```diff
@@ -54,6 +54,8 @@ def table_partial(workbasket)
       "workbaskets/shared/steps/review_and_submit/edit_footnote"
     elsif workbasket.type == 'edit_regulation'
       "workbaskets/shared/steps/review_and_submit/edit_regulation"
+    elsif workbasket.type == 'create_certificate'
+      "workbaskets/shared/steps/review_and_submit/create_certificate"
     end
   end
```
Comment (steveholmes) [inline app/helpers/cross_check_helper.rb]: Does the cross_check_title method need a new elsif as weel?

File: app/views/workbaskets/create_certificate/show.html.slim [hunk 3]
```diff
@@ -1,30 +1,26 @@
...
+  h1.heading-large
+    | View certificate
+
+  .view-workbasket-container
+    = render "workbaskets/create_certificate/workflow_screens_parts/notifications/block", screen_type: :view
+    = render "workbaskets/create_certificate/workflow_screens_parts/actions_allowed"
+    = render "workbaskets/create_certificate/workflow_screens_parts/workbasket_details"
+
+    - if iam_workbasket_author? && workbasket_rejected?
+      = link_to "Edit cer", '#',
```
Comment (steveholmes) [inline app/views/workbaskets/create_certificate/show.html.slim]: Is this meant to be "Edit cer"?

**Comment by KyleMacPherson** [comment]
@steveholmes thanks, nice spots!

**Pull request merged**